src/nylas-connection.js

```diff
@@ -159,6 +159,9 @@ module.exports = class NylasConnection {
           }
           if (response.statusCode) {
             error.statusCode = response.statusCode;
+            if (!error.hasOwnProperty('message')) {
```

Comment:
Hi @Tranced, I think we may want to also check `typeof response.body === 'string'` to make sure we're returning a string. It's also worth considering adding an `else` clause in case `response.body` isn't a string, and just responding with a generic error message.

Reply:
Sounds good to me! I'm not sure what cases we'll run into if the error message is not a string. So, in that case, I'll just leave it with the current default behavior.